test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -57,6 +57,15 @@ void TestBackward(
   }
 }
 
+TEST_F(AtenXlaTensorTest, TestClone) {
+  ForEachDevice([&](const Device& device) {
+    at::Tensor a = at::rand({2, 2}, at::TensorOptions(at::kFloat));
+    at::Tensor xla_a = bridge::CreateXlaTensor(a, device);
+    at::Tensor xla_b = xla_a.clone();
```

Comment:
Let's do an in-place operation on `a` and `xla_a` to prove that this is indeed a clone.

Reply:
SG